tests/Command/DumpEnvCommandTest.php

```diff
@@ -164,6 +164,46 @@ public function testDoesNotRequireToSpecifyEnvArgumentWhenLocalFileIsPresent()
         unlink($envLocalPhp);
     }
 
+    public function testLoadLocalEnvWhenTestEnvIsNotEqual()
+    {
+        @mkdir(FLEX_TEST_DIR);
+        $env = FLEX_TEST_DIR.'/.env';
+        $envLocal = FLEX_TEST_DIR.'/.env.local';
+        $envLocalPhp = FLEX_TEST_DIR.'/.env.local.php';
+        $composer = __DIR__.'/../../composer.json';
+        @unlink($envLocalPhp);
+
+        file_put_contents($env, 'APP_ENV=dev');
+        $envContent = <<<EOF
+APP_ENV=test
+APP_SECRET=abcdefgh123456789
+EOF;
+        file_put_contents($envLocal, $envContent);
+
+        copy($composer, FLEX_TEST_DIR.'/composer-backup.json');
+        $composerContent = @json_decode(file_get_contents($composer), true);
+        $composerContent['extra']['runtime']['test_envs'] = [];
```

Comment:
I think this test case tests nothing right now.
I order for it to do anything, a `.env.test` file should be added to me.

Reply:
oh nvm, `.env.local` is the one